gradle.properties

```diff
@@ -4,10 +4,6 @@ org.gradle.parallel=true
 org.gradle.caching=true
 # mandatory from AGP 3.6
 android.useAndroidX=true
-# The next version of all artifacts
-# Scheme: year.month[.<version>]
-# Version is just a consecutive number. We don't need to bound releases to days.
-projectVersion=2020.5.6
 # Current stable version to be used in this project
 infraVersion=2020.5.5
```

Comment:
Do we need to fix [publishing instructions](https://avito-tech.github.io/avito-android/docs/infrastructure/#publishing-a-new-release)?
I didn't get when to change and publish a new version

Reply:
Done